questions/qa-address-formats.en.md

```diff
@@ -0,0 +1,107 @@
+# Address formats around the world
+
+## Question
+
+### How do address formats differ around the world, and what are the implications of those differences on the design of forms, databases, ontologies, etc. for the Web?
+
+Address formats vary widely across the globe, with differences in structure, content, and the level of granularity. For authors and developers designing forms, databases, or systems that handle addresses, understanding these variations is crucial to avoid frustrating users from other countries. This article will introduce some of the key differences in address formats around the world and provide guidance on how to design systems that can handle them effectively.
```

Comment:
I think it would be good to include lots of visual example, pretty much for each point you make, rather than just the few you have.

Reply:
Thanks, I added more examples.